phlib/basesup.c

```diff
@@ -598,6 +598,7 @@ SIZE_T PhCountStringZ(
     _In_ PWSTR String
     )
 {
+#ifndef _ARM64_
     if (PhpVectorLevel >= PH_VECTOR_LEVEL_SSE2)
```

Comment:
Is it easier to set PhpVectorLevel to zero than ifdef the code?

Reply:
Nope. Because the SSE X86 intrinsics are not available for ARM, so compiler error occurs.